keyboards/keychron/c3_pro/c3_pro.c

```diff
@@ -16,6 +16,40 @@
 
 #include "c3_pro.h"
 
+void keyboard_post_init_kb(void) {
+    setPinOutputPushPull(LED_MAC_OS_PIN);
+    setPinOutputPushPull(LED_WIN_OS_PIN);
+    writePin(LED_MAC_OS_PIN, !LED_OS_PIN_ON_STATE);
+    writePin(LED_WIN_OS_PIN, !LED_OS_PIN_ON_STATE);
+    
+    keyboard_post_init_user();
+}
+
+void eeconfig_init_kb(void) {
+#if (EECONFIG_KB_DATA_SIZE) == 0
+    // Reset Keyboard EEPROM value to blank, rather than to a set value
+    eeconfig_update_kb(0);
+#endif
+    keymap_config.raw  = eeconfig_read_keymap();
+    keymap_config.nkro = 1;
+    eeconfig_update_keymap(keymap_config.raw);
+
+    eeconfig_init_user();
+}
```

Comment:
I would rather we didnt introduce this (especially as its not consistent with the rest of the keychron boards)
```suggestion
```